.eslintrc

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "airbnb",
+  "env": {
+    "mocha": true
+  },
+  "rules": {
+    "semi": [2, "never"],
+    "no-param-reassign": 0
```

Comment:
I'd like to have this rule enabled, there's some particular reason to disable it?